index.html

```diff
@@ -5,6 +5,7 @@
     <link rel="icon" type="image/svg+xml" href="/static/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Raleway:500,700,800&display=swap"/>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Courier+Prime:400&display=swap"/>
```

Comment:
I recall it being possible to combine these into a single resource, might be good to do.